<launch>

    <!-- Configurable arguments -->
    <arg name="robot_name" default="sam"/>

	<!-- topics, note the root! -->
    
    <node name="emergency_surface_action" pkg="sam_action_servers" type="emergency_surface_action.py" output="screen" ns="$(arg robot_name)/ctrl">
	    <param name="emergency_topic" value="/$(arg robot_name)/core/abort" />
        <param name="vbs_cmd_topic" value="/$(arg robot_name)/core/vbs_cmd" />
        <param name="rpm1_cmd_topic" value="/$(arg robot_name)/core/thruster1_cmd" />
        <param name="rpm2_cmd_topic" value="/$(arg robot_name)/core/thruster2_cmd" />
	</node>

</launch>
